cmd/run.go

```diff
@@ -390,12 +391,42 @@ func Run(isDebug *bool) *cli.Command {
 }
 
 func printErrorsInResults(errorsInTaskResults []*scheduler.TaskExecutionResult, s *scheduler.Scheduler) {
-	errorPrinter.Printf("\nFailed tasks: %d\n", len(errorsInTaskResults))
-	for _, t := range errorsInTaskResults {
-		errorPrinter.Printf("  - %s\n", t.Instance.GetHumanReadableDescription())
-		errorPrinter.Printf("    └── %s\n\n", t.Error.Error())
+	errorPrinter.Printf("\nFailed assets: %d\n", len(errorsInTaskResults))
+	data := make(map[string][]*scheduler.TaskExecutionResult)
+
+	for _, result := range errorsInTaskResults {
+		assetName := strings.Split(result.Instance.GetHumanID(), ":")[0]
+		data[assetName] = append(data[assetName], result)
 	}
 
+	tree := treeprint.New()
+	columnBranches := make(map[string]treeprint.Tree)
+
+	for assetName, results := range data {
+		branch := tree.AddBranch(assetName)
+		for _, result := range results {
+			parts := strings.Split(result.Instance.GetHumanID(), ":")
```

Comment:
these instances all have their own types, let's use those instead: https://github.com/bruin-data/bruin/blob/main/pkg/scheduler/scheduler.go#L152

Reply:
@karakanb Can you please explain it ?